tao_compiler/mlir/disc/transforms/lhlo_legalize_roots_to_loops.cc

```diff
@@ -2044,6 +2156,11 @@ LogicalResult lowerWithScheduleLoopCPU(
                                  parallel_loop, shape_constraint_analysis);
   }
 
+  if (root_ops.size() == 1 && isLargeConcatOp(root_ops[0])) {
+    return lowerWithScheduleLargeConcatCPU(
+        root_ops, dominant_op, parent, non_fusion, shape_constraint_analysis);
+  }
+
```

Comment:
This is more like a brand new schedule for fusion codegen. Will it be better to explicitly add a new mlir::disc_ral::FusionType? instead of more branches in loopCPU schedule.

Reply:
done